community/cypher/expressions-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/expressions/ArithmeticExpressions.scala

```diff
@@ -96,6 +98,9 @@ case class Divide(lhs: Expression, rhs: Expression)(val position: InputPosition)
     TypeSignature(argumentTypes = Vector(CTInteger, CTInteger), outputType = CTInteger),
     TypeSignature(argumentTypes = Vector(CTInteger, CTFloat), outputType = CTFloat),
     TypeSignature(argumentTypes = Vector(CTFloat, CTFloat), outputType = CTFloat),
+    TypeSignature(argumentTypes = Vector(CTFloat, CTInteger), outputType = CTFloat),
+    TypeSignature(argumentTypes = Vector(CTFloat, CTDuration), outputType = CTDuration),
```

Comment:
That is not correct. You should be able to divide a duration by a number, but _not_ vice versa.